app/assets/stylesheets/components/common/_global_nav.scss

```diff
@@ -0,0 +1,206 @@
+  transition-duration: 1s;
+
+  position: absolute;
+  left: 0;
+  transform: translateX(0%);
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  background: $color-yellow-light;
+  font-size: 1rem;
+
+  @include respond-to($mq-m) {
+    @include column(12);
+
+    transition: none;
+    transform: none;
+    position: static;
+    background: transparent;
+    display: inline-block;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+  }
+
+  &.is-active {
+    transform: translateX(-100%);
+
+    @include respond-to($mq-m) {
+      transform: none;
+    }
+  }
+}
+
+.global-nav__clump {
+  border-bottom: solid 1px rgba(0, 0, 0, 0.375);
+  box-shadow: inset 0 -1px 2px 0 rgba(0, 0, 0, 0.375);
+  position: static;
+  margin: 0;
+  line-height: 1;
+
+  @include respond-to($mq-m) {
+    border-bottom: none;
+    box-shadow: none;
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+  }
+
+  &.is-active {
+    .global-nav__content {
+      transform: translateX(100%);
+
+      @include respond-to($mq-m) {
+        display: block;
+        transform: none;
+      }
+    }
+
+    @include respond-to($mq-m) {
+      background: $color-white;
+    }
+  }
+}
+
+.global-nav__clump:last-child {
+  border-bottom: none;
+}
+
+.global-nav__clump:first-child {
+  border-top: none;
+
+  @include respond-to($mq-m) {
+    .global-nav__clump__heading__text:before {
+      border-left: none;
+    }
+  }
+}
+
+.global-nav__clump__heading {
+  @include column(12);
+
+  display: inline-block;
+  float: none;
+  line-height: 25px;
+  padding: $baseline-unit*2 0;
+  border: none;
+  background: transparent;
+  text-align: left;
+
+  @include respond-to($mq-m) {
+    position: relative;
+    display: flex;
+    width: 100%;
+    margin: 0;
+    line-height: 1;
+    text-align: center;
+    flex-grow: 1;
+
+    &:hover {
+      background: $color-white;
+
+      & + .global-nav__content {
+        display: block;
+      }
+    }
+  }
+}
+
+.global-nav__clump__blog-link {
+  background: $color-mas-blog-blue;
+  border-radius: 4px;
+  color: $color-white;
+  font-weight: 700;
+  padding: $baseline-unit/2 $baseline-unit*2.5;
+  text-transform: uppercase;
+  width: auto;
+  margin-top: $baseline-unit;
+  margin-bottom: $baseline-unit;
+
+  .global-nav__arrow {
+    display: none;
+  }
+
+  @include respond-to($mq-m) {
+    margin-left: $baseline-unit;
+    margin-right: $baseline-unit;
+
+    &:hover {
+      background: $color-mas-blog-blue-hover;
+    }
+  }
+}
+
+.global-nav__clump__heading__text {
+  @include respond-to($mq-m) {
+    padding: 0 $baseline-unit;
+    flex-grow: 1;
+
+    &:before {
+      content: "";
+      position: absolute;
+      top: $baseline-unit*2;
+      bottom: $baseline-unit*2;
+      left: -0.5px;
+      border-left: 1px solid;
+      width: 1px;
+    }
+  }
+}
+
+.global-nav__clump__blog-link__text {
+  padding: $baseline-unit*0.5 0;
```

Comment:
I mentioned this to the Cultivate guys recently, so I'll repeat myself here (sorry) - dividing up the baseline unit makes it pointless using the baseline unit - think it would be simpler to just use an non-variable value here?